source/topics/paas/how-to-modify-nginx-config.md

```diff
@@ -34,18 +31,22 @@ app configuration to the NGiNX proxies:
   when using these settings as syntax errors in these settings can keep your
   NGiNX instance from starting.
 
+Setting these variables is a two-step process:
+
+1. Set the app variable.
+2. Run `aptible restart` to update the endpoint with your changes.
+
 Either or both of `FORCE_SSL` and `DISABLE_WEAK_CIPHER_SUITES` can be enabled
 by setting the environment variables to `true` on the corresponding Aptible app
 using the [Aptible CLI](/topics/cli/how-to-install-cli):
 
 ```
-aptible config:set FORCE_SSL=true DISABLE_WEAK_CIPHER_SUITES=true --app $APP_HANDLE
+aptible config:set FORCE_SSL=true DISABLE_WEAK_CIPHER_SUITES=true --app $APP_HANDLE && aptible restart --app $APP_HANDLE
```

Comment:
I think we should include it here. However, I think we should either make this two lines or include an escaped line break: right now our code blocks require horizontal scrolling if they're longer than a single line, which could make this easy to miss.